practicals/large_language_models.ipynb

```diff
@@ -16,7 +16,7 @@
         "\n",
```

Comment:
<p>Add a note here that if people are not familiar with <code>einsum</code> to see the numpy documentation: https://numpy.org/doc/stable/reference/generated/numpy.einsum.html </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='136'/>

Reply:
<p>Added in </p>